### PR TITLE
Optional events

### PR DESCRIPTION
**Pull request by gmcquillan**
- Added Optional Events Support
- Added Supervisorctl Support
- Added Extra Entrypoints.

### CODE REVIEW

File: firetower/admin.py [hunk 1]
```diff
@@ -120,7 +126,7 @@ def main():
 
     conf = config.Config(options.conf_path)
     admin = Admin(conf)
-    wait_seconds = options.delay
+    wait_seconds = options.delay * 60
```
Comment (dukesam) [inline firetower/admin.py:129]: so options.delay is measured in 1/60s of a second?

File: firetower/category.py [hunk 2]
```diff
@@ -170,21 +170,24 @@ def __init__(self, redis_conn, cat_id):
...
+    def add_event(self, event, timestamp=None,
+            event_increment=True, ts_increment=True):
         """Add an event to the event list.
 
         Args:
             event: the event dictionary to save into the event set
             timestamp: Optional time stamp to use when inserting the event.
                 Defaults to now.
-            ts_increment: Optional. Increments the counter for this category.
+            event_increment: Optional. boolean - adds event to event series.
```
Comment (dukesam) [inline firetower/category.py:181]: +1

**Comment by dukesam** [comment]
:fire:

File: firetower/admin.py (hunk 1 above)
Comment (gmcquillan) [inline firetower/admin.py:129]: Nah. The commanline option is minutes between cycles. So we convert minutes to seconds for time.sleep() here.

**Pull request merged**